### PR TITLE
feature: venue setup invite assignment

### PR DESCRIPTION
**Pull request by celestemartinez**

### CODE REVIEW

File: openreview/venue/process/invite_assignment_post_process.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
+def process(client, edge, invitation):
+
+    SHORT_PHRASE = ''
```
Comment (melisabok) [inline openreview/venue/process/invite_assignment_post_process.py:3]: we should try reading the variables from the domain object content.

File: openreview/venue/process/simple_paper_recruitment_process.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
+def process(client, note, invitation):
+    from Crypto.Hash import HMAC, SHA256
+    import urllib.parse
+    from datetime import datetime
+    VENUE_ID = ''
```
Comment (melisabok) [inline openreview/venue/process/simple_paper_recruitment_process.py:5]: do we need this file? can we read the variable from the domain group?

Comment (celestemartinez) [reply openreview/venue/process/simple_paper_recruitment_process.py:5]: yes, we can, I haven't gotten around to updating this yet

File: openreview/venue/webfield/areachairsWebfield.js [hunk 3]
```diff
@@ -2,7 +2,28 @@
 const reviewerAssignmentTitle = null
 const areaChairsId = domain.content.area_chairs_id.value
 const reviewerGroup = domain.content.reviewers_id.value
-// Add bid invitation if enabled
```
Comment (melisabok) [inline openreview/venue/webfield/areachairsWebfield.js:5]: @xkopenreview please review

**Comment by melisabok** [comment]
@xkopenreview @zbialecki I wonder if there is a nicer way to show this, this is a paper recruitment invitation, we don't have the info of the paper, only the paper id. 

Screen Shot 2023-02-08 at 9 08 37 PM

**Comment by zbialecki** [comment]
@melisabok I'm not sure what you're suggesting we show here, are the submission notes even visible to non-authenticated users?

I do think we could remove the "Please see venue website" text

**Review by melisabok — APPROVED** [review]

**Pull request merged**